arch/arm64/configs/xiaomi-tissot.config

```diff
@@ -1,9 +1,11 @@
 CONFIG_BACKLIGHT_QCOM_WLED=y
 CONFIG_BATTERY_QCOM_FG=m
+CONFIG_CHARGER_QCOM_SMBCHG=m
```

Comment:
Recheck how battery working, this should be built-in

Reply:
Not necessary. Charger and fg just need to be in the same state